libbeat/docs/tab-widgets/start.asciidoc

```diff
@@ -1,17 +1,20 @@
 // tag::deb[]
+
+:beatname_url: {beats-ref-root}/{beatname_lc}/{branch}
```

Comment:
This can't be added here as it doesn't resolve for the attributes that are used in tagged regions elsewhere. This needs to be added everywhere that attribute is used.

@dedemorton, I know you were against putting this in the docs repo initially (and I was too after hearing your argument), but maybe we can put it in the docs repo with a big comment around it saying not to use it for external book linking? I'm struggling to think of a better option

```suggestion
:beatname_url: {beats-ref-root}/{beatname_lc}/{branch}
```

Reply:
I think it *should* work because the `beatname_url` attribute is only referenced in the widgets right? 

At any rate, I can see that including it in the widget adds a LOT of repetition, so I think you're probably right. Put it in with the other shared attributes and mark it clearly. 

@EamonnTP Sorry to give you the run around on this, but I think Brandon is right.

Reply:
As it turns out, @dedemorton, you were right 😄